resources/resources/software/aws/toolkits/resources/localized_messages.properties

```diff
@@ -251,6 +251,7 @@ serverless.application.deploy.error.invalid_runtime_group=Cannot find runtime gr
 serverless.application.deploy.error.unsupported_runtime_group=The runtime {0} is not supported for deploying the SAM template {1}
 serverless.application.deploy.error.no_resources=Cannot find any resources in SAM template {0}
 serverless.application.deploy.error.empty_runtime=The runtime is not defined for all functions in template {0}
+serverless.application.deploy.error.bad_parse=Error parsing SAM template {0}\n\n {1}
```

Comment:
double new line??

Reply:
That was intentional, actually that reminded me that I forgot to attach a screenshot of it.